### PR TITLE
FOGL-3326_2: added SETENV: to sudoer to allow sudo -E apt -y upgrade

### PR DESCRIPTION
**Pull request by sstefano69**
Signed-off-by: Stefano Simonelli <stefano@dianomic.com>

the sudo does not permits to export variables using the current cfg 

```
export DEBIAN_FRONTEND=noninteractive
sudo -E apt -y upgrade
sudo: sorry, you are not allowed to preserve the environment
```

we can add 'SETENV: ' to support this.

*DO YOU SEE ANY ISSUES OR SECURITY IMPLICATION ?*

### CODE REVIEW

**Review by MarkRiddoch — DISMISSED** [review]

**Review by pintomax — DISMISSED** [review]

**Review by praveen-garg — DISMISSED** [review]
We need to document it, the SETENV thing is needed in existing install of 1.7.0 sudoers file as well before upgrade is happening. 

including  DEBIAN_FRONTEND=noninteractive needs. this looks important for the users having docker based installs / automated install / upgrade mechanisms.

**Comment by sstefano69** [comment]
> We need to document it, the SETENV thing is needed in existing install of 1.7.0 sudoers file as well before upgrade is happening.
> 
> including DEBIAN_FRONTEND=noninteractive needs. this looks important for the users having docker based installs / automated install / upgrade mechanisms.

@praveen-garg  - @MarkRiddoch 
I have added the section 'Upgrading FogLAMP on Ubuntu' into README.rst to provide the information related to the DEBIAN_FRONTEND.

**Review by praveen-garg — DISMISSED** [review]

File: README.rst [hunk 1]
```diff
@@ -149,7 +149,28 @@ The destination may be overriden by setting the variable *DESTDIR* in the make c
 
 |br|
 
+Upgrading FogLAMP on Ubuntu
```
Comment (praveen-garg) [inline README.rst]: `Ubuntu` -> `Debian based systems`

File: README.rst [hunk 2]
```diff
@@ -149,7 +149,28 @@ The destination may be overriden by setting the variable *DESTDIR* in the make c
...
+
+	# for Kerberos authentication, avoid interactive questions
+	DEBIAN_FRONTEND=noninteractive apt install -yq krb5-user
+	DEBIAN_FRONTEND=noninteractive apt install -yq libcurl4-openssl-dev
+
+The upgrade of the FogLAMP package should follow the same philosophy, it should be done executing the command:
+::
+    sudo DEBIAN_FRONTEND=noninteractive apt -y upgrade
+
+before the upgrade of FogLAMP, *SETENV:* should be set/added in */etc/sudoers.d/<user>* to allow *sudo* to support the handling of the environment variables, a sample of the file:
```
Comment (praveen-garg) [inline README.rst]: can we simply saye `<user>` -> `*/etc/sudoers.d/foglamp`

File: README.rst (hunk 1 above)
Comment (sstefano69) [reply README.rst]: done

File: README.rst (hunk 2 above)
Comment (sstefano69) [reply README.rst]: done

**Review by praveen-garg — DISMISSED** [review]

**Review by praveen-garg — APPROVED** [review]

**Review by MarkRiddoch — APPROVED** [review]

**Pull request merged**